src/parser/ValueParser.java

```diff
@@ -0,0 +1,46 @@
+package parser;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ValueParser {
+    private static ValueParser valueParser;
+    private Object value;
+
+    private ValueParser() {
+    }
+
+    public static ValueParser getInstance() {
+        if (valueParser == null) {
+            valueParser = new ValueParser();
+        }
+
+        return valueParser;
+    }
+
+    public Object getValue(String valueName, String stringValue) {
+        switch (valueName) {
+            case "Id":
+                Pattern idPattern = Pattern.compile(" Id :(.)+");
+                Matcher idMatcher = idPattern.matcher(stringValue);
+
+                if (idMatcher.find()) {
```

Comment:
In other method you didn't use if statement when find() method is used. Try to do this there.